packages/client/src/service/txpool.ts

```diff
@@ -379,6 +398,26 @@ export class TxPool {
     }
   }
 
+  pruneBlobsAndProofsCache() {
+    // currently common.param errors when not achieved cancun hardfork to do the below calculation
+    // const blobGasLimit = this.config.chainCommon.param('maxblobGasPerBlock')
+    // const blobGasPerBlob = this.config.chainCommon.param('blobGasPerBlob')
+    //
+    // A better way could be figured out to get allowedBlobsPerBlock
+    const allowedBlobsPerBlock = 6
```

Comment:
Why can't we pull this from common? It does not make sense to call this method pre-Cancun right?

Reply:
somehow its called from some tests, will need to debug